Libplanet.Tests/Action/AccountStateDeltaTest.cs

```diff
@@ -40,16 +40,16 @@ protected AccountStateDeltaTest(ITestOutputHelper output)
 
             _addr = _keys.Select(AddressExtensions.ToAddress).ToArray();
 
-#pragma warning disable CS0612
+#pragma warning disable CS0618
             _currencies = new[]
             {
-                Currency.LegacyUntracked("FOO", 0, _addr[0]),
-                Currency.LegacyUntracked("BAR", 0, _addr.Take(2).ToImmutableHashSet()),
-                Currency.LegacyUntracked("BAZ", 0, null),
+                Currency.Legacy("FOO", 0, _addr[0]),
+                Currency.Legacy("BAR", 0, _addr.Take(2).ToImmutableHashSet()),
+                Currency.Legacy("BAZ", 0, null),
                 Currency.Uncapped("QUX", 0, minter: _addr[0]),
                 Currency.Capped("QUUX", 0, (100, 0), minter: _addr[0]),
             };
-#pragma warning restore CS0612
+#pragma warning restore CS0618
```

Comment:
Could you leave a short comment to justify this suppression?

Also, the scope to suppress CS0618 could be narrower, as only `Currency.Legacy()` is obsolete.